test/devices_list/devices_bloc_picking_device.dart

```diff
@@ -0,0 +1,49 @@
+
+import 'package:mockito/mockito.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:test/test.dart';
+import 'package:wear_hint/devices_list/devices_bloc.dart';
+import 'package:wear_hint/model/ble_device.dart';
+import 'package:wear_hint/state/application_state.dart';
+
+import '../factory/factory.dart';
+import '../mocks/mocks.dart';
+
+void main() {
+  FlutterBlueMock flutterBlueMock;
+  DeviceRepositoryMock deviceRepositoryMock;
+  DevicesBloc devicesBloc;
+  BleDevice bleDevice;
+
+  setUp(() {
```

Comment:
Jakoś to duże `U` dziwnie mi tu wygląda. Zrobiłbym po prostu `setup`.

Reply:
To jest wywołanie funkcjii `setUp` a funkcja ta pochodzi z frameworku do testów jednostkowych